gnucash/import-export/aqb/gnc-flicker-gui.c

```diff
@@ -0,0 +1,517 @@
+/*
+ * gnc-flicker-gui.c --
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of
+ * the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, contact:
+ *
+ * Free Software Foundation           Voice:  +1-617-542-5942
+ * 51 Franklin Street, Fifth Floor    Fax:    +1-617-542-2652
+ * Boston, MA  02110-1301,  USA       gnu@gnu.org
+ */
+
+/**
+ * @internal
+ * @file gnc-flicker-gui.c
+ * @brief GUI callbacks for Flicker and ChipTAN(optisch)
+ * @author Copyright (C) 2020 Christian Wehling <christian.wehling@web.de>
+ */
+
+#include <config.h>
+
+#include <gtk/gtk.h>
+#include <glib/gi18n.h>
+
+#include "dialog-utils.h"
+#include "gnc-flicker-gui.h"
+#include "gnc-state.h"
+#include "gnc-ui.h"
+
+#define GNC_PREFS_GROUP "dialogs.flicker"
+#define GNC_STATE_SECTION "Flicker"
+#define STATE_KEY_BARWIDTH "barwidth"
+#define STATE_KEY_DELAY "delay"
```

Comment:
This is just cosmetic but I would line up on "barwidth"

Reply:
I don't understand what you mean by the hint. "barwidth" is the width of the individual bars in the flicker display.

Reply:
```
#define GNC_STATE_SECTION  "Flicker"
#define STATE_KEY_BARWIDTH "barwidth"
#define STATE_KEY_DELAY    "delay"

```